.github/actions/remove-version-prefix/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Remove Version Prefix
+
+inputs:
+  version: 
+    type: string
+
+outputs:
+  without_prefix:
+    value: ${{ steps.action.outputs.version }}
+    
+runs:
+  using: "composite"
+  steps:
+    - name: Remove it
+      shell: bash
+      id: action
+      run: |
+        # remove the "v" prefix.
+        version=$(echo "${{ inputs.version }}" | tr -d 'v')
+
+        # update the package.json file with the current version.
```

Comment:
**Issue:** I think this comment is wrong.

Maybe we can update it to something like `Adds the version to output`

Reply:
Oops, done c13cd57dcd8c858cb3007278eba6058edaf06394